lib/utils/config.js

```diff
@@ -97,14 +97,14 @@ module.exports.bootstrap = function bootstrap(serverConfig, clientConfig, mode)
 
 /**
  * @param {String} filePath
- * @param {{app:string, client:string}} config
+ * @param {{app:string, server: string, client:string}} config
  * @returns {{app:string, server:string, client:string}}
  */
 module.exports.appConfig = function appConfig(filePath, config) {
-    if (config && config.app && config.client) {
+    if (config && config.app && config.server && config.client) {
         return {
             app: config.app,
-            server: config.app,
+            server: config.server,
```

Comment:
Check first if config.server exists then use it. 

Server: config.server ? config.server : config.app

Sorry for formatting I’m on my phone.

Reply:
No problem. Thank you for the quick review.